### ISSUE TITLE
run hello_world in Docker got 'numpy._DTypeMeta' object is not subscriptable

### ISSUE THREAD

**Issue by weiminw** [princeton-vl/infinigen#163]
# Describe the bug

### What version of the code were you using? 
'''
What is the output of `git log -n 1`?
'''

### What command did you run?
```
$BLENDER -noaudio --background --python generate.py -- --seed 0 --task coarse -g desert.gin simple.gin --output_folder outputs/hello_world/coarse
```

### What are your FULL output logs?
```
Blender 3.3.1 (hash b292cfe5a936 built 2022-10-05 00:14:35)
Error: Python: Traceback (most recent call last):
  File "/opt/infinigen/worldgen/generate.py", line 20, in <module>
    from terrain import Terrain
  File "/opt/infinigen/worldgen/terrain/__init__.py", line 7, in <module>
    from .core import Terrain
  File "/opt/infinigen/worldgen/terrain/core.py", line 16, in <module>
    from terrain.scene import scene, transfer_scene_info
  File "/opt/infinigen/worldgen/terrain/scene.py", line 10, in <module>
    from terrain.elements.landtiles import LandTiles, Volcanos, FloatingIce
  File "/opt/infinigen/worldgen/terrain/elements/landtiles.py", line 12, in <module>
    from terrain.assets.landtiles import assets_to_data, landtile_asset
  File "/opt/infinigen/worldgen/terrain/assets/landtiles/__init__.py", line 7, in <module>
    from .core import assets_to_data, landtile_asset
  File "/opt/infinigen/worldgen/terrain/assets/landtiles/core.py", line 15, in <module>
    from .ant_landscape import ant_landscape_asset
  File "/opt/infinigen/worldgen/terrain/assets/landtiles/ant_landscape.py", line 13, in <module>
    from terrain.land_process.snowfall import run_snowfall
  File "/opt/infinigen/worldgen/terrain/land_process/snowfall.py", line 10, in <module>
    from landlab import RasterModelGrid
  File "/opt/infinigen/blender/3.3/python/lib/python3.10/site-packages/landlab/__init__.py", line 17, in <module>
    from .core.model_component import Component
  File "/opt/infinigen/blender/3.3/python/lib/python3.10/site-packages/landlab/core/model_component.py", line 36, in <module>
    from ..field import FieldError
  File "/opt/infinigen/blender/3.3/python/lib/python3.10/site-packages/landlab/field/__init__.py", line 2, in <module>
    from .graph_field import GraphFields
  File "/opt/infinigen/blender/3.3/python/lib/python3.10/site-packages/landlab/field/graph_field.py", line 2, in <module>
    import xarray as xr
  File "/opt/infinigen/blender/3.3/python/lib/python3.10/site-packages/xarray/__init__.py", line 3, in <module>
    from xarray import testing, tutorial
  File "/opt/infinigen/blender/3.3/python/lib/python3.10/site-packages/xarray/testing.py", line 11, in <module>
    from xarray.core.coordinates import Coordinates
  File "/opt/infinigen/blender/3.3/python/lib/python3.10/site-packages/xarray/core/coordinates.py", line 16, in <module>
    from xarray.core.alignment import Aligner
  File "/opt/infinigen/blender/3.3/python/lib/python3.10/site-packages/xarray/core/alignment.py", line 24, in <module>
    from xarray.core.variable import Variable, as_compatible_data, calculate_dimensions
  File "/opt/infinigen/blender/3.3/python/lib/python3.10/site-packages/xarray/core/variable.py", line 48, in <module>
    from xarray.namedarray.core import NamedArray
  File "/opt/infinigen/blender/3.3/python/lib/python3.10/site-packages/xarray/namedarray/core.py", line 24, in <module>
    from xarray.namedarray._typing import (
  File "/opt/infinigen/blender/3.3/python/lib/python3.10/site-packages/xarray/namedarray/_typing.py", line 28, in <module>
    _DType = TypeVar("_DType", bound=np.dtype[Any])
TypeError: 'numpy._DTypeMeta' object is not subscriptable
```

### Platform
- OS & OS Version:  Ubuntu-22.04
- GPU (RTX 4090) :
- GPU Driver Version (?) :
- RAM (GB): 128

# Additional context

**Comment by MalcDuck** [comment]
Having the EXACT same issue here.   No Errors in install log.  And I believe i followed ALL The steps to install correctly.

But as the above user didnt attach their install script, here's mine:

[InfinigenDuck.txt](https://github.com/princeton-vl/infinigen/files/13199960/InfinigenDuck.txt)

And as it always seems to be asked :

wsl -l -v
  NAME            STATE           VERSION
* Ubuntu-22.04    Running         2

**Comment by MalcDuck** [comment]
Hi Weiminw

Just FYI - I fixed this in my code by adding : 

`xarray==2023.9.0`

to requirements.txt, as the latest version breaks the code.

Then run 

`bash install.sh 
`
again, and all should be well (or it was for me)

**Comment by araistrick** [comment]
Thanks for the fix @MalcDuck, either I can patch this onto main or id happily accept a PR. We experienced this before in #156 where an update to scipy also broke compatibility with our numpy version. Our numpy is relatively old but im surprised this keeps popping up. I could just pin our entire requirements.txt to specific versions but I feel this is a little restrictive. On our `rc_1.1.1` preview branch I have updated our code to work with the latest numpy version but that branch still has some visual bugs, I may try to separate out and backport this fix.

**Comment by Khoa-NT** [comment]
> Hi Weiminw
> 
> Just FYI - I fixed this in my code by adding :
> 
> `xarray==2023.9.0`
> 
> to requirements.txt, as the latest version breaks the code.
> 
> Then run
> 
> `bash install.sh ` again, and all should be well (or it was for me)

I can confirm that this works for my M2 MBA

**Comment by weiminw** [comment]
> Hi Weiminw
> 
> Just FYI - I fixed this in my code by adding : 
> 
> `xarray==2023.9.0`
> 
> to requirements.txt, as the latest version breaks the code.
> 
> Then run 
> 
> `bash install.sh 
> `
> again, and all should be well (or it was for me)

great thanks to you，i upgrade the numpy to 1.23.5，error is gone

**Comment by araistrick** [comment]
Hello all, thanks for reporting this issue, this should now be fixed by #176 on main.
